pkg/kubectl/resource_printer.go

```diff
@@ -1748,6 +1752,63 @@ func printClusterRoleBindingList(list *rbac.ClusterRoleBindingList, w io.Writer,
 	return nil
 }
 
+func printCertificateSigningRequest(csr *certificates.CertificateSigningRequest, w io.Writer, options PrintOptions) error {
+	meta := csr.ObjectMeta
+	name := meta.Name
+	kind := options.KindName
+
+	if options.WithKind {
+		name = kind + "/" + name
+	}
+
+	if _, err := fmt.Fprintf(
+		w, "%s\t%s",
+		name,
+		translateTimestamp(meta.CreationTimestamp),
+	); err != nil {
+		return err
+	}
+	var approved, denied bool
```

Comment:
do you think the requesting user and whether or not a cert has been issued should be in the printer, or should they have to switch to the describer to see that?